src/lcm/entry_point.py

```diff
@@ -21,7 +22,7 @@
 
 
 def get_lcm_function(
```

Comment:
I made the type hint changes and updated the docstring. I won't go much further, because the entry point will be completely rewritten in the next 1-2 months.